cmd/upgrade.go

```diff
@@ -0,0 +1,218 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var upgradeYes bool
+
+var upgradeCmd = &cobra.Command{
+	Use:   "upgrade",
+	Short: "Keep a terraform template up to date",
+	Run:   doUpgrade,
+	Example: `
+fargate-create upgrade
+fargate-create upgrade -t git@github.com:turnerlabs/terraform-ecs-fargate-scheduled-task
+fargate-create upgrade -d infrastructure
+`,
+}
+
+func init() {
+	rootCmd.AddCommand(upgradeCmd)
+}
+
+func doUpgrade(cmd *cobra.Command, args []string) {
+
+	//has the user done a previous install?
+	notFound := false
+	if _, err := os.Stat(targetDir); err != nil {
+		notFound = true
```

Comment:
Here's how I read this.  If a stat of the target directory succeeds (meaning it exists -- we found it), then we set notFound=true and later we error out with "no existing template found".

Reply:
Nevermind, I got that backwards.  It seems this check could be omitted though.  If targetDir doesn't exist, neither will Join(targetDir, baseDir) which is the next check.

Reply:
Good point.  I'll remove the extraneous check.